### PR TITLE
Gor 0.13.0

### PR DESCRIPTION
**Pull request by buger**
- [ X ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [ X ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ X ] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [ X ] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/gor.rb [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 class Gor < Formula
   desc "Real-time HTTP traffic replay tool written in Go"
-  homepage "https://github.com/buger/gor/"
-  url "https://github.com/buger/gor/archive/v0.10.1.tar.gz"
-  sha256 "283ca037a782844df42a0352c072efb558ffca3dc76f88a6317eca0d44ab1a5c"
-  head "https://github.com/buger/gor.git"
```
Comment (DomT4) [inline Formula/gor.rb]: Why the removal of `head`?

File: Formula/gor.rb [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
+  homepage "https://gortool.com"
+  url "https://github.com/buger/gor.git",
+      :tag => "v0.13.0",
+      :revision => "98fe7f19b21eaaffcd0d733a57668f63de5921d7"
```
Comment (MikeMcQuaid) [inline Formula/gor.rb]: Why move to a git checkout, out of interest? Submodules?

Comment (buger) [inline Formula/gor.rb]: Using Git is way easier for me and makes more sense then computing sha of downloaded file, and frankly I borrowed this technique from `packer` formula.

Does it have disadvantages?

File: Formula/gor.rb (hunk 1 above)
Comment (buger) [inline Formula/gor.rb]: Good point, rolled it back

**Comment by buger** [comment]
Thanks for the comments, I tried to address them.

File: Formula/gor.rb (hunk 2 above)
Comment (MikeMcQuaid) [inline Formula/gor.rb]: Yes, it's significantly slower for users and more expensive for GitHub. Please move it back to the old way, thanks.

**Comment by buger** [comment]
Thank you for the review! Now it use files with sha256 as before.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**